common/source/docs/common-cuav-c-rtk2.rst

```diff
@@ -0,0 +1,201 @@
+   - Active CW detection and removal Onboard band pass ﬂter
+- Support flight control type
+   -  Compatible with flight controllers running ArduPilot/PX4 firmware
+- Interfaces
+   - 1 Hotshoe
+   - 1 Shutter in
+   - 1 Sutter out
+   - 1 Type C(HS_USB)
+   - 1 F9P USB
+   - 1 F9P UART
+   - 1 Antenna(mmcx)
+- Supply voltage
+   - 4.5~6v
+- Operating temperature
+   - -20~85℃
+- Size
+   - 56x33x16.5mm
+- Weight
+   -39g
+
+Purchase
+========
+
+`CUAV Alibaba <https://cuav.en.alibaba.com/product/1600431150057-820872629/Free_shipping_Pre_sale_price_CUAV_C_RTK_2_Muti_frequency_PPK_Drone_PPK_Kit_for_Pixhawk_Fixed_Wing_VTOL.html?spm=a2700.shop_index.11406.1.194479958nMdw6)>`__
+
+`CUAV Store <https://store.cuav.net>`__
+
+`CUAV Aliexpress <https://pt.aliexpress.com/item/1005003754165772.html?spm=a2g0o.store_pc_home.slider_6001949835246.0>`__
+
+LED Indicators
+==============
+
+.. image:: ../../../images/c-rtk2/led.png
+	:target: ../images/c-rtk2/led.jpg
+
+- SYS Led
+   - 100ms flashing: In bootloader
+   - 1000ms flashing: Normal working condition
+- TF LED(TF Card）
+   - Flashing: Noraml SD card reading and writing
+   - Always bright: No TF card or file system error of TF card. Try cycling the power.
+- Shutter In led (on side)
+   - Flashing： Camera shutter signal input
+- Shutter Out led
+   - Flashing： Output shutter trigger signal to camera
+- CAN LED
+   - Dark： No data input and output
+   - Blink once every 2 seconds： Data is being sent, but no data is beingreceived
+   - Blink twice within 2 seconds: Data is being received, but no data is being sent
+   - Blink 3 times in 2 seconds: Sending and Receiving data.
+- PPS LED
+   - On: no FIX
+   - Flashing: FIX
+- RTK LED
+   - Blinking 3 times every 2 seconds: Sending and Receiving RTK data.
+   - Flash：RTK Float
+   - Solid On：RTK FIX
+   - Dark：NO RTK
+- RUN LED
+   - Red led always On： System Error
+   - Yellow led always On： Application Warning
+   - Yellow flashing (100ms)： GNSS(F9P) Firmware update
+   - Blue Flashing: Base station mode
+   - Green Flashing: Rover mode
+- Mode button
+   - When not connected to the computer, press and hold for 3 seconds to switch the mode
+
+Pinouts
+=======
+
+.. image:: ../../../images/c-rtk2/c-rtk2_pinouts1.jpg
+	:target: ../images/c-rtk2/c-rtk2_pinouts1.jpg
+
+.. image:: ../../../images/c-rtk2/c-rtk2_pinouts0.jpg
+	:target: ../images/c-rtk2/c-rtk2_pinouts0.jpg
+
+.. image:: ../../../images/c-rtk2/c-rtk2_pinouts2.jpg
+	:target: ../images/c-rtk2/c-rtk2_pinouts2.jpg
+
+Preparation before use
+======================
+Only use PPK
+------------
+
+.. image:: ../../../images/c-rtk2/ppk1.jpg
+	:target: ../images/c-rtk2/ppk1.jpg
+
+- 1 PPK base station/Ntrip account 
+- 1 C-RTK 2 1
+- 1 Mapping camera 
+- 1 Drone 
+
+PPK+RTK
+-------
+
+.. image:: ../../../images/c-rtk2/rtk1.jpg
+	:target: ../images/c-rtk2/rtk1.jpg
+
+- 1 PPK base station/Ntrip account 
+- 1 C-RTK 2 
+- 1 Mapping camera 
+- 1 Drone 
+- 1 Data link 
+- 1 Computer 
+
+.. note::
```

Comment:
I do not understand this note...in the PPK figure you are using this as a PPK base station, why is it not recommended? what exactly are you trying to say?

Reply:
C-RTK2 BASE is another product under development; C-RTK2 is used as a PPK base station; it first needs to know the absolute precise position of the current point (latitude and longitude and the height of the pole); and it is not conducive to users Measure control points.

Reply:
Here is the configuration tutorial for the base station: https://doc.cuav.net/gps/c-rtk2/en/base-station.html; can you help me add some needed information?

Reply:
I want to make the docs clear to a naive user like myself...I dont understand a lot about PPK and RTK setups....but here is what I think I know, correct me if I am wrong:

 For PPK, you need the raw GPS data from the vehicle (always C-RTK2 in this case), and either an historical data base of RTK data or the RTK data from a calibrated PPK base station. The "calibrated" can be exact GPS coordinates of the base entered into the base so it can generate RTK data, or an RTK base that has "surveyed in" itself over time to establish its exact position. Apparently the C-RTK2 does not store the PPK  data in a base station friendly format or does not have the ability to survey itself, or cannot accept entered GPS location, and that is why its not recommended for PPK Base use.

Also, the labels on the two figures are confusing...we need to use exact naming to avoid confusion, so I propose that a third diagram be added, the the current ones modified as below. I would avoid labels as AIR unless you actually call the C-RTK2 AIR, especially since it can be on any vehicle type:
PPK
![ppk1](https://user-images.githubusercontent.com/6076534/153643296-a8d177bd-d59c-49b8-b93b-0251a5071989.jpg)
RTK
![rtk2](https://user-images.githubusercontent.com/6076534/153644317-7ac2c5c7-3079-4a15-a706-bbaacd135617.jpg)
PPK + RTK
![rtk1](https://user-images.githubusercontent.com/6076534/153644090-7f4e6b32-5e40-4d83-9910-5f497097b564.jpg)

let me know if my understanding is incorrect

Reply:
I recreated the picture and sent it to your skype;